main.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+from src.model import preprocessing, corpus_counts, bag_of_words, tf_idf, mlb, evaluation
+
+def main():
+
+	train = preprocessing.read_data('./data/train.tsv')
```

Comment:
For the structure don't we want this main file into separate files like the sms-spam-detection is done? Might also be easier for the dvc

Reply:
Do we save the model to file anywhere after training? For inference we simply load the model from the file

Reply:
Yeah agree, but I think the nicest way to do it is to merge this and then fix it in the DVC PR